cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -2109,6 +2109,53 @@ action.cyences_notable_event_action.attacker_search = | stats count by LogonErro
 action.cyences_notable_event_action.attacker_drilldown = `cs_o365` sourcetype="o365:management:activity" Workload=AzureActiveDirectory Operation=UserLoginFailed LogonError=$row.LogonError$
 action.cyences_send_email_action = 1
 action.cyences_notable_event_action.products = Office 365
+action.cyences_notable_event_action.deprecated = 1
+action.cyences_notable_event_action.deprecated_from_version = 4.0.0
+action.cyences_notable_event_action.deprecated_replacement = O365 - Login Failure From New Location Due To Multi Factor Authentication
+
+[O365 - Login Failure From New Location Due To Multi Factor Authentication]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 9,39 * * * *
+description = This alert will show the login failure from new location due to multi factor authentication. \
+\
+Data Collection - Office 365 management activity data (Splunk Add-on for Office 365). 
+dispatch.earliest_time = -2h@h
+dispatch.latest_time = +2h@h
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_o365` sourcetype="o365:management:activity" _index_earliest=-31m@m _index_latest=-1m@m Workload=AzureActiveDirectory Operation=UserLoginFailed (LogonError="DeviceAuthenticationRequired" OR LogonError="UserStrongAuthClientAuthNRequiredInterrupt") \
+| eval ExtendedProperties=mvzip('ExtendedProperties{}.Name','ExtendedProperties{}.Value'," : ") \
+| stats count, values(user_type) as user_type, values(authentication_method) as authentication_method, max(_time) as Last_Failed_Login, values(LogonError) as LogonError, values(ExtendedProperties) as ExtendedProperties by user, ClientIP, app \
+| iplocation ClientIP \
+| lookup cs_authentication_usual_location.csv app, user, Country OUTPUT pct \
+| fillnull value=0 pct \
+| where pct < `cs_authentication_new_location_login_pct_limit` \
+| fillnull Country, Region, City value="-" \
+| eval Location=ClientIP." (".count.") | ".Country." | ".Region." | ".City \
+| stats sum(count) as count, values(pct) as pct, values(user_type) as user_type, values(authentication_method) as authentication_method, max(Last_Failed_Login) as Last_Failed_Login, list(Location) as Location, values(LogonError) as LogonError, values(ExtendedProperties) as ExtendedProperties by user \
+| eval cyences_severity = case(pct==0, "high", user_type=="Admin" or user_type=="DcAdmin", "medium", true(), "low") \
```

Comment:
should it be critical for pct=0.
Also, check you are doing values(pct) as pct so, it may create problem. with the condition.